react-stopwatch-component/src/stopwatch.jsx

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import PlayIcon from './play-icon';
+
+export default class StopWatch extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      isPaused: true,
+      seconds: 0,
+      timer: 0
```

Comment:
If you change `timer:` to `null`, you can remove your `isPaused` state and just use whether `this.state.timer` is null or not to determine the run state